src/applications/vaos/appointment-list/redux/selectors.js

```diff
@@ -267,6 +267,7 @@ export function getConfirmedAppointmentDetailsInfo(state, id) {
 
 export function getPastAppointmentListInfo(state) {
   return {
+    showScheduleButton: selectFeatureRequests(state),
```

Comment:
This was missing, so the schedule link wasn't showing up for the past appointments page